packages/assets-api/README.md

```diff
@@ -0,0 +1,66 @@
+# @talend/assets-api
+
```

Comment:
This maybe requires more context than the requirements. 

What real-life issues is it trying to solve? Can you give example? Is it replacing something? As a newbie I don't really understand the point.

Reply:
Agree with that, and it's missing a description on top to tell what it does. The documentation starts with requirements but we don't know requirements for what.

Reply:
let me know if this is better now